pkg/addons/data/ConfigMap/openshift-node/node-config-compute.yaml

```diff
@@ -36,6 +36,10 @@ data:
       - /etc/origin/node/pods
       rotate-certificates:
       - "true"
+      kube-reserved:
```

Comment:
should we do this on masters as well?

Reply:
Master does not run any user workload, and kubelet static pods are the ones with heavy workload. Not sure if it worth limiting those (never did this on side for those specific reasons). Maybe @kwoodson can tell how they do in Dedicated?